src/main/java/com/serenitydojo/calculator/Calculator.java

```diff
@@ -1,7 +1,68 @@
 package com.serenitydojo.calculator;
 
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+
 public class Calculator {
+
+    List<String> items = new ArrayList<>();
+
     public int evaluate(String expression) {
-        return 0;
+        int evaluvatedValue=0;
+
+        if (!(expression.isEmpty())){
+            items = Arrays.stream(expression.split(" "))
+                    .filter(val->!(val.isEmpty()))
+                    .collect(Collectors.toList());
+            evaluvatedValue = calculateValue(items);
+        }
+
+        return evaluvatedValue;
+
     }
+
+
+    private int calculateValue(List<String> strItems){
+
+        int itemCount = strItems.size();
+        int i=0 ;
+        List<Integer> numbers=new ArrayList<>();
+        String operator ="";
+        while (i<itemCount){
+            String val = strItems.get(i);
+            if(val.chars().allMatch(Character::isDigit)){
+                numbers.add(Integer.valueOf(val));
+            }else{
+                operator  = val;
+            }
+            i++;
+            if (numbers.size()==2 && !operator.isEmpty()){
+                int resultValue =0 ;
+                if (operator.equals("+")){
+                    resultValue = numbers.get(0) + numbers.get(1);
+                }else if (operator.equals("-")){
+                    resultValue = numbers.get(0) -numbers.get(1);
+                }else if(operator.equals("*")){
+                    resultValue = numbers.get(0) *numbers.get(1);
+                }else if(operator.equals("/")){
+                    if(numbers.get(1)!=0) {
+                        resultValue = numbers.get(0) / numbers.get(1);
+                    }else{
+                        throw new IllegalMathsOperatorException("Division by zero");
```

Comment:
was that a required behavior?  
it isn't really an IllegalMathsOperator, `/` is legal. And java will throw it's own DevisionByZeroException, no need to catch that in my opinion

Reply:
It was not part of the requirement mentioned in the challenge (division all together ) . I thought to add it as it looks more complete with all 4 arithmetic operations. I removed the division by zero test